packages/dev/core/src/Meshes/abstractMesh.ts

```diff
@@ -61,6 +61,9 @@ function applyMorph(data: FloatArray, kind: string, morphTargetManager: MorphTar
         case VertexBuffer.UVKind:
             getTargetData = (target) => target.getUVs();
             break;
+        case VertexBuffer.ColorKind:

```

Comment:
Not related to your PR, but could you also add the `UV2Kind` case, as it seems we forgot it?

Thanks in advance!